_posts/2024-08-07-accessing-kafka-with-gateway-api.md

```diff
@@ -0,0 +1,675 @@
+---
+layout: post
+title: "Accessing Kafka with the Gateway API"
+date: 2024-08-07
+author: colt_mcnealy
+---
+
+At [LittleHorse](https://littlehorse.dev), we use the [Gateway API](https://gateway-api.sigs.k8s.io) to allow external traffic into our kubernetes clusters. We will soon need to allow external clients to access our [Kafka](https://kafka.apache.org) clusters (managed by Strimzi, of course!) from outside of our Kubernetes clusters. We have so far been pleased with the performance and simplicity of [Envoy Gateway](https://gateway.envoyproxy.io/) as a Gateway Controller, which motivated this investigation into using Envoy Gateway to access our Kafka clusters.
+
+### Background
+
+The [Gateway API](https://gateway-api.sigs.k8s.io/) in Kubernetes aims to replace the `Ingress` resource as the de facto standard for allowing external traffic to reach workloads running on Kubernetes. It addresses many shortcomings of the `Ingress` resource, including poor support for non-HTTP 1.0 traffic.
+
+![Strimzi and Envoy Gateway](/assets/images/posts/2024-08-07-strimzi-gateway-api.png)
```

Comment:
I would go with same format as you use for Strimzi. Without background (same for Kafka) it will look much better form my POV. Also maybe use a little bit better resolution? It will look better for sure